README.md

```diff
@@ -378,6 +388,24 @@ pub fn evaluate_member(&self, ctx: ExecCtx, ...) -> StdResult<Response> {
 }
 ```
 
+
+## Executor message builder
+
+Execute messages of other contracts can be built with `Remote` as well by
+calling `executor` method. It returns a message builder that implements
+auto-generated `Executor` traits of all sylvia contracts.
```

Comment:
Since the `ExecutorBuilder` is generic over a contract this type implements only the traits of this contract and all the interfaces it implements.

I might over expand on this for README, but feel free to consider some parts.

```suggestion
Sylvia defines the [`ExecutorBuilder`]() type, which can be accessed through [`Remote::executor`]().
It's generic over the contract type and exposes execute methods from the contract and every interface implemented on it through an auto-generated `Executor` traits.
Execute messages of other contracts can be built with `Remote` as well by
calling `executor` method. It returns a message builder that implements
auto-generated `Executor` traits of all Sylvia contracts.

Methods defined in the `Executor` traits constructs an execute message, which variant corresponds to the method name.
The message is then wrapped in the `WasmMsg`, and returned once [`ExecutorBuilder::build()`]() method is called.
```